collection_requirements.rst

```diff
@@ -186,6 +186,7 @@ Also:
 
 * change M(), examples, seealso, extended_documentation_fragments to use actual FQCNs (in moved content and in other collections that have references to the content)
 * move all related issues / pull requests / wiki pages
+* look through ``docs/docsite`` directory of `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using ``grep`` command-line utility) to check if there are examples using the moved modules / plugins to actualize their FQCNs
```

Comment:
```suggestion
* look through ``docs/docsite`` directory of `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using the ``grep`` command-line utility) to check if there are examples using the moved modules / plugins to update their FQCNs
```

Reply:
in my language `actualize` == `update`. Update also works here

Reply:
In mine too, but not in English :)

Reply:
https://en.wiktionary.org/wiki/actualize

Reply:
cool, maybe it came from german originally, thanks for clarifying:)